cdap-data-fabric/src/main/java/io/cdap/cdap/data2/metadata/lineage/field/FieldLineageInfo.java

```diff
@@ -321,6 +338,12 @@ private long computeChecksum() {
                                      operationsMap.get(input.getOrigin()), write, summary);
       }
     }
+    for (TransformOperation transform : dropTransforms) {
+      for (InputField input : transform.getInputs()) {
+        Operation previous = operationsMap.get(input.getOrigin());
+        computeIncomingSummaryHelper(NULL_EPF, transform, previous, summary);
```

Comment:
here it should be `computeIncomingSummaryHelper(NULL_EPF, previous, transform, summary);`. the transform will become previous in the method.